driverkit/config/dev/flatcar_stable_4.19.106-flatcar.yaml

```diff
@@ -0,0 +1,5 @@
+kernelrelease: 4.19.106-flatcar
+target: flatcar-stable
```

Comment:
We'd need this target in driverkit to merge this PR :)

Reply:
Yep, I'll make sure to wire a PR that too.